p2p/src/network_ext.rs

```diff
@@ -8,13 +8,42 @@ use bitcoin::{Network, TestnetVersion};
 
 use crate::Magic;
 
+const BITCOIN_SEEDS: &[&str] = &[
+    "seed.bitcoin.sipa.be",
+    "dnsseed.bluematt.me",
+    "dnsseed.bitcoin.dashjr-list-of-p2p-nodes.us",
+    "seed.bitcoin.jonasschnelli.ch",
+    "seed.btc.petertodd.net",
+    "seed.bitcoin.sprovoost.nl",
+    "dnsseed.emzy.de",
+    "seed.bitcoin.wiz.biz",
+    "seed.mainnet.achownodes.xyz",
+];
```

Comment:
Out of interest, why do the strings in C++ strings end with a period?

A few questions around code comments:
- Since the Core devs thought it meaningful to list the persons name , is there some reason you thought not to?
- Is the stuff about service bits useful?

Reply:
`.` (dot) is the root domain.  All other top-level domains descend from dot, like .com. .net. .org (think of a tree).  Technically it's `google.com.` but nobody bother with the last `.` since it's implied.  The wiki on root name server talks a bit about it if you're still curious.

Reply:
I guess dot is supposed to signify the empty string at the top of the hierarchy but a dot is put in place to signify full-stop actually.

Reply:
It might be worth it though in this case to be overly pedantic and follow suit adding the dot (period)

Reply:
Cool, TIL. Thanks